src/Umbraco.Core/Persistence/SqlSyntax/SqlServerSyntaxProvider.cs

```diff
@@ -237,6 +241,44 @@ public override bool TryGetDefaultConstraint(IDatabase db, string tableName, str
             return !constraintName.IsNullOrWhiteSpace();
         }
 
+        public override void WriteLock(IDatabase db, int? writeLockReasonId, params int[] lockIds)
+        {
+            // soon as we get Database, a transaction is started
+
+            if (db.Transaction.IsolationLevel < IsolationLevel.ReadCommitted)
+                throw new InvalidOperationException("A transaction with minimum RepeatableRead isolation level is required.");
+
+
+            // *not* using a unique 'WHERE IN' query here because the *order* of lockIds is important to avoid deadlocks
+            foreach (var lockId in lockIds)
+            {
+                db.Execute(@"SET LOCK_TIMEOUT 1800;");
```

Comment:
Need constants here

Reply:
For the "1800" or what do you want as constant?